<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabShMove">
        <attribute name="title">
         <string>Shadow movement</string>
        </attribute>
        <widget class="QPushButton" name="pushButtonStartSim">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>260</y>
           <width>91</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Simulate</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>190</y>
           <width>57</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>210</y>
           <width>81</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabDayEvol">
        <attribute name="title">
         <string>Daylight evolution </string>
        </attribute>
        <widget class="QWidget" name="widgetDatEvolCond" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>431</width>
           <height>41</height>
          </rect>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>0</y>
            <width>431</width>
            <height>41</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxSumTime">
             <property name="text">
              <string>Summer time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxRestrCond">
             <property name="text">
              <string>Day between 8:00 and 20:00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxTimeChange">
             <property name="text">
              <string>Change time</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QPushButton" name="pushButtonCalculate">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>280</y>
           <width>90</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>160</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>75</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelGnomHeight">
       <property name="text">
        <string>Gnomon height, m</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTimeZone">
       <property name="text">
        <string>Time zone</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditHeight">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxZone">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxCoords">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Gnomon coordinates</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <item row="1" column="2">
       <widget class="QLabel" name="labelLatGrad_2">
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="labelLongSec">
        <property name="text">
         <string>'</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labelLatSec">
        <property name="text">
         <string>'</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelLat">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelLatGrad">
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditLongSec"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditLongGrad"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditLatGrad"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditLatSec"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLong">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
